### ISSUE TITLE
The mu-law encoding overflows occasionally

### ISSUE THREAD

**Issue by cbquillen** [ibab/tensorflow-wavenet#209]
I tried changing the tf.nn.softmax_cross_entropy_with_logits() for the loss to use tf.nn.sparse_softmax_cross_entropy_with_logits() and then got NaNs.  The problem turned out to be encoded mu-law values > quantization_levels.  It turns out that librosa's load (in audio_reader.py) can produce audio with |audio| > 1.   This isn't all that rare (every 20 chunks or so.)

**Comment by veqtor** [comment]
Could we normalise the audio file using some librosa function?
It's likely that sample-rate conversion causes these types of problems.
